### ISSUE TITLE
TS2305: Module '"type-fest"' has no exported member 'Jsonifiable'

### ISSUE THREAD

**Issue by timdream** [block65/webcrypto-web-push#16]
```
> tsc -p web-push-proxy/tsconfig.json

node_modules/@block65/webcrypto-web-push/dist/lib/types.d.ts:1:15 - error TS2305: Module '"type-fest"' has no exported member 'Jsonifiable'.

1 import type { Jsonifiable, RequireAtLeastOne } from 'type-fest';
                ~~~~~~~~~~~
```

Since this package does not define `type-fest` as a dependency directly, `npm` seems to decide to install `type-fest@2` as a dependency of `serialize-error`. Only the later versions of `type-fest` exports `Jsonifiable`.

I can easily fix this by `npm i --save-dev type-fest` but we can improve this package by explicitly require `type-fest@4` as a dependency. Thanks!

**Comment by maxholman** [comment]
Couldnt replicate this, but I can see why it might be happening.

`type-fest` is in `devDependencies" of the web-push package, so does not get installed by the package that depends on it, hence types are either unavailable or imported from elsewhere, as in this case.

Fix incoming

**Comment by maxholman** [comment]
Published as v1.0.2
